compiler/passes/checkResolved.cpp

```diff
@@ -115,6 +115,46 @@ static void checkSyncSingleAtomicDefaultInit() {
 }
 
 
+// This function is checking for any function which returns a a non-default
+// copyable type (sync/single/atomic). This has exclusions for functions that
+// are no copying like array aliases or explicit "no copy" functions. This
```

Comment:
Maybe "for functions that don't copy"?  Or "actions that don't copy, like array aliases"?